scss/ui/_autosuggest.scss

```diff
@@ -0,0 +1,82 @@
+/*------------- MIXINS -----------------*/
+// Transition
+@mixin transition($args) {
+  -webkit-transition: $args;
+  -moz-transition: $args;
+  -ms-transition: $args;
+  -o-transition: $args;
+  transition: $args;
+}
+
+//Border Raduis
+@mixin border-radius($radius) {
+  -webkit-border-radius: $radius;
+  -moz-border-radius: $radius;
+  -ms-border-radius: $radius;
+  border-radius: $radius;
+}
+
+//Box Shadow
+@mixin box-shadow($shadow) {
+  -webkit-box-shadow: $shadow;
+  box-shadow: $shadow;
+  -moz-box-shadow: $shadow;
+  -o-box-shadow: $shadow;
+  border-collapse: separate;
+}
+
+//Box Shadow None
+@mixin box-shadow-none{
+  -webkit-box-shadow: none;
+  -moz-box-shadow: none;
+  -ms-box-shadow: none;
+  box-shadow: none;
+}
+.fd-suggestions {
+  margin: 20px 0 0;
+  max-height: calc(100vh - 130px);
+  overflow-y: auto;
+  @include box-shadow(0 1px 1px 0 rgba(0, 0, 0, 0.2));
+  .fd-suggestion-item {
+    padding: 20px;
+    border-bottom: 1px solid #E7E3E3;
+    background: #FFFFFF;
```

Comment:
Can you please add colors as variables. Also we follow alphabetical order for CSS properties, could you also please change it